### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by AnkitKumarvaid**
last line updated

### CODE REVIEW

**Review by littleironical — CHANGES_REQUESTED** [review]
this didn't excite me, please do some good changes.

File: README.md [hunk 1]
```diff
@@ -72,3 +72,4 @@ classroom evaluation.
 
 ## Support
 If you like this project, don't forget to give it a ⭐
+####################################################################################################################################
```
Comment (littleironical) [inline README.md:75]: remove this line

Comment (AnkitKumarvaid) [reply README.md:75]: ok

**Comment by littleironical** [comment]
Closing this PR as it was just for testing

**Pull request closed without merging**